docking.js

```diff
@@ -1636,6 +1668,7 @@ var DockManager = new Lang.Class({
     Name: 'DashToDock.DockManager',
 
     _init: function() {
+        this._remoteModel = new LauncherAPI.LauncherEntryRemoteModel();
```

Comment:
I don't know that it is strictly necessary, but this object is never destroyed.

In Line ~1921 we can add `this._remoteModel.destroy();`, since the `.destroy()` method is already available.